views/partials/_toc-patterns.njk

```diff
@@ -1,43 +1,22 @@
 <nav class="app-c-toc">
```

Comment:
The patterns and component 'toc' files should be combined back into one and the layout changed (or even removed) from the patterns and components. We only needed two separate files whilst the navigation was being manually created.

Also, I've just realised that TOC is not really the right name. It made sense for the tech docs, where the left hand nav _was_ a table of contents for the current page, but not now that it's a proper sub navigation. It might make sense to address that at the same time, though equally we could address it in a future PR if you prefer.

Reply:
now using a single layout and renamed `toc` to `subnav` as it makes more sense